frontend/src/components/Posts/FilterContainer.tsx

```diff
@@ -139,6 +139,19 @@ export const FilterContainer = ({
     );
   };
 
+  // don't show "requests" and "offers" filter on unverified org user's "my posts" tab
```

Comment:
is this the part that's not working atm?

Reply:
Yes

Reply:
So this code should handle the rendering of these filter selections, not preventing people from creating a new post.
![image](https://github.com/uaem-na/donations-tracker/assets/8529473/13cdee1f-8e34-4168-b78d-a7b8d48a05eb)

Reply:
Instead, we should disable these buttons here:
![image](https://github.com/uaem-na/donations-tracker/assets/8529473/4a70caa3-5896-446f-b638-6ec417851eb5)

And then prevent unverified organization accounts to access these routes:
{base}/posts/offer/new
{base}/posts/request/new